specs/experimental/admin-and-auth.md

```diff
@@ -25,21 +28,131 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-The Security Council (at
-[eth:0xc2819DC788505Aac350142A7A707BF9D03E3Bd03](https://etherscan.io/address/0xc2819DC788505Aac350142A7A707BF9D03E3Bd03))
-uses a specially extended Safe multisig contract to provide additional security guarantees on top of
-those provided by the Safe contract.
+## Overview
+
+Authorization of administrative actions in the Superchain are managed by a system of multisignature
+ Safe contracts, as well as custom extensions to those Safe contracts which provide additional
+functionality. This document describes the system of Safe's and their purposes.
+
+## List of Safes
+
+This list outlines the various Safes, including mainnet addresses, thresholds, ownership, and any
```

Comment:
Should specs include addresses? They aren't technically part of the spec and I don't think we list them anywhere else in the spec (excluding e.g. predeploys). Maybe we talk with:
-  @tessr about adding them to the superchain registry? I think they would fit here: https://github.com/ethereum-optimism/superchain-registry/blob/4810f97b7ee9407e383761c00ff89856c33d6156/superchain/configs/mainnet/superchain.toml#L2-L3
- @sbvegan about adding them to the docs?

Reply:
They should be in the docs, ideally the privileged roles doc gets refactored to be privileged roles in the Superchain. All members of the superchain should be included by pulling that information from the superchain registry